packages/wonder-blocks-button/package.json

```diff
@@ -1,8 +1,11 @@
 {
   "private": true,
-  "name": "wonder-blocks-button",
+  "name": "@khanacademy/wonder-blocks-button",
   "version": "0.0.1",
   "design": "v1",
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
This is needed so that the packages we publish are public.

Reply:
It avoids having to re-authenticate which is impossible when running `lerna`.  You still need to do `npm login` before running `yarn run publish`.